circle.yml

```diff
@@ -0,0 +1,35 @@
+machine:
+  environment:
+    GOVERSION: 1.6
+    GOPATH: /home/ubuntu/go_workspace
+    GOROOT: /home/ubuntu/go$GOVERSION
+    PATH: /home/ubuntu/google-cloud-sdk/bin:/home/ubuntu/go$GOVERSION/bin:$GOPATH/bin:$PATH
+  python:
+    version: 2.7.3
+  services:
+    - docker
+
+dependencies:
+  cache_directories:
+    - ../google-cloud-sdk
+    - ../go_workspace
+    - ../go1.6
```

Comment:
I was thinking about this with the other projects we are using this go-circle magic on ... 
can this use the $GOVERSION env var so it doesn't have to be duplicated? Alternatively we could make our install-go.sh script for circle install $GOROOT as /home/ubuntu/go instead of go$VERSION.